client/src/components/auth/RegistrationForm.js

```diff
@@ -28,7 +32,15 @@
         setInputs({...inputs, [name]: value})
     };
 
-    const [validated, setValidated] = useState(false);
+    const registrationResult = (isSuccessful, errorMessage) => {
```

Comment:
I realised something about this: passing registrationResult function to the Redux action is not recommended: https://redux.js.org/style-guide/style-guide#reducers-must-not-have-side-effects

Reply:
Interesting read. I couldn't find in the guide whether it says not to use callbacks inside Redux action. They recommend not doing it in Reducers though. Regardless, I searched around and I think I found a better way to do it. Thanks for this